analyses/molecular-subtyping-HGG/01-HGG-molecular-subtyping-defining-lesions.Rmd

```diff
@@ -53,9 +53,10 @@ if (!dir.exists(results_dir)) {
 
 # Read in metadata
 metadata <-
-  readr::read_tsv(file.path(root_dir, "data", "pbta-histologies-base.tsv"), 
-                  guess_max = 10000) %>%
-  dplyr::filter(sample_type == "Tumor") 
+  readr::read_tsv(file.path(root_dir, "data", "histologies-base.tsv"), 
+                  guess_max = 100000) %>%
+  dplyr::filter(sample_type == "Tumor") %>% 
+  dplyr::filter(cohort == "PBTA")
```

Comment:
@kgaonkar6, good point! The reason for doing this is because currently TP53 score is only run on PBTA cohort - previously we added TARGET and GMKF in but the ROC cure does not look that great. They all fall into ~50% so I am just add this temporarily here.

Reply:
Do you think it's possible to maybe not add a filter here but to only use the logic to add tp53 scores?

Reply:
So I added this:
```
tp53_status <- read_tsv(file.path(
  root_dir,
  "analyses",
  "tp53_nf1_score",
  "results",
  "tp53_altered_status.tsv"), guess_max = 100000)

# Filter metadata to contain only samples that have TP53 scores
tp53_sample_id <- tp53_status %>% 
  pull(sample_id) %>% unique()

metadata <- metadata %>% 
  filter(sample_id %in% tp53_sample_id)
```
Is this what you meant?

Reply:
I meant if TP53 score is the only issue the https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/hgg-pull-update/analyses/molecular-subtyping-HGG/10-HGG-TP53-annotation.Rmd file should be updated. What would the reason be to filter for PBTA in other scripts, do we want to only subtype PBTA samples?

Reply:
Oh since the final result will have either tp53 activated or loss annotated, if we do not do that, then we cannot be sure whether the fact that the lack of annotation if due to tp53 is normal or there is no TP53 normal

Reply:
Then I think the logic of handling missing gistic results for WXS will be applicable here?

Reply:
But then in the final table - we will need to have another category saying "TP53, not available`? I actually think it is easier to leave it as `PBTA` for now (or use all the samples that currently have TP53 score) and once we figure out the scores for GMKF and TARGET, we can simply remove this and re-run. 
Additionally, we are moving towards using a database method to do subtyping and currently only the new PNOC samples need to run through the module and get subtypes, so I don't think we want to re-invent the wheel and add an additional category for the final table.

Reply:
Ok approving the code changes here since there are other plans for subtyping.